en/mkdocs.yml

```diff
@@ -652,6 +653,7 @@ plugins:
         'administer/product-administration/monitoring/monitoring-with-opentracing.md': 'https://apim.docs.wso2.com/en/3.1.0/administer/logging-and-monitoring/monitoring/monitoring-with-opentracing/'
         'administer/product-administration/monitoring/capturing-system-data-in-error-situations.md': 'https://apim.docs.wso2.com/en/3.1.0/troubleshooting/capturing-system-data-in-error-situations/'
         'administer/product-administration/monitoring/jmx-based-monitoring.md': 'https://apim.docs.wso2.com/en/3.1.0/administer/logging-and-monitoring/monitoring/jmx-based-monitoring/'
+        'administer/product-administration/monitoring/monitoring-server-health.md': 'https://apim.docs.wso2.com/en/3.1.0/administer/logging-and-monitoring/monitoring/monitoring-server-health/'
```

Comment:
This redirection does not seem to be needed.

Reply:
Removed with 3f8b87b